src/integration_tests/basset_vault/definition.ts

```diff
@@ -657,7 +646,11 @@ async function calculate_repay_cycles_amount(result: BlockTxBroadcastResult) {
         if (action_1 == "repay_loan") {
             repay_cycles_amount += 1;
         }
-
     }
     return repay_cycles_amount;
+}
+
+async function assert_numbers_with_inaccuracy(expected: number, actual: number, inaccuracy: number) {
```

Comment:
this function should be `sync` (without `async` prefix), and when you use it - remove `await`

Reply:
done